src/main/java/com/github/u3games/eventengine/EventEngineManager.java

```diff
@@ -278,6 +280,8 @@ public void listenerOnLogout(L2PcInstance player)
 		{
 			if (_state == EventEngineState.REGISTER || _state == EventEngineState.VOTING)
 			{
+				L2GameClient client = player.getClient();
+				DualBoxProtection.getInstance().removeConnection(client);

```

Comment:
Just use single line: ```DualBoxProtection.getInstance().removeConnection(player.getClient());```
Or add a method that handles a player and gets the client.

Btw, the client could be null?